.env.example

```
@@ -10,5 +10,10 @@ HUGO_THEME_DIR=../Fried-Rice
 # Flask 密钥（生产环境必须修改）
 SECRET_KEY=dev-secret-key-change-in-production
 
+# 管理员账户（首次启动时写入 data/auth.json，之后可在应用内修改密码）
+# 未设置时默认 用户名=admin 密码=admin（仅用于本地，请尽快修改）
+ADMIN_USERNAME=admin
+ADMIN_PASSWORD=admin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ADMIN_PASSWORD key should go before the ADMIN_USERNAME key

(UnorderedKey)